src/main/java/com/nanuvem/metagui/server/controller/OperationalController.java

```diff
@@ -0,0 +1,72 @@
+package com.nanuvem.metagui.server.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import com.google.gson.Gson;
+import com.nanuvem.metagui.server.api.EntityType;
+import com.nanuvem.metagui.server.container.DomainModelContainer;
+import com.nanuvem.metagui.server.container.EntityTypeDomain;
+
+@Controller
+@RequestMapping(value = "/api/{resource}")
+public class OperationalController {
```

Comment:
Split this code into, OperationalController and OperationalService.
You do not need to fix all comments now. Create a minor issue for each one and we will fix them later.